c2corg_ui/static/js/documentediting.js

```diff
@@ -374,6 +389,25 @@ app.DocumentEditingController.prototype.handleMapFeatureChange_ = function(
 
 
 /**
+ * It doesn't associate, it just pushes it into associations array.
+ * @private
+ */
+app.DocumentEditingController.prototype.pushDocToAssociations_ = function() {
```

Comment:
I specified that I don't want to 'really' associate the doc, I just push it into the array of the not-yet-existing doc. Maybe I don't know how it works behind, on the API side.
If you tell me that a virtual document ID is created and even at this point I can already associate the existing doc to this non existing, then you are right - I could use apiservice.associateDocument